lib/AbiBinProvider.js

```diff
@@ -22,18 +21,32 @@ class OpenSTAbiBinProvider extends AbiBinProvider {
   /**
```

Comment:
I am not sure why OpenSTAbiBinProvider is inheriting from AbiBinProvider of mosaic?

Reply:
OpenST is on top of mosaic. Inheriting  AbiBinProvider of mosaic helps end users to retrieve abi/bin of mosaic with a single OpenST AbiBinProvider object.

Reply:
But should they need to? Why?